### ISSUE TITLE
Mock Service default works only with VSS 3.1.1

### ISSUE THREAD

**Issue by lukasmittag** [eclipse/kuksa.val.services#82]
Mock Service default mock.py works only with the VSS 3.1.1 VSS 4.0 creates the following error:
```
ERROR:base_service:Failed to register datapoints
Traceback (most recent call last):
  File "/home/imu6fe/kuksa.val.services/.venv/lib/python3.8/site-packages/lib/baseservice.py", line 106, in _on_broker_connectivity_change
    self.on_databroker_connected()
  File "mockservice.py", line 74, in on_databroker_connected
    loader_result = PythonDslLoader().load(self._vdb_metadata)
  File "/home/imu6fe/kuksa.val.services/.venv/lib/python3.8/site-packages/lib/loader.py", line 140, in load
    mocked_datapoints = self._load_mocked_datapoints(vdb_metadata)
  File "/home/imu6fe/kuksa.val.services/.venv/lib/python3.8/site-packages/lib/loader.py", line 65, in _load_mocked_datapoints
    metadata = vdb_metadata[datapoint["path"]]
KeyError: 'Vehicle.Cabin.Seat.Row1.Pos1.Position'
```

**Comment by lukasmittag** [comment]
After further investigation the only thing thats not working with VSS 4.0 are the examples in mock.py. iw ould suggest we write this in the README and solve this issue through this. PR will follow shortly.

**Comment by SebastianSchildt** [comment]
will be fixed by #95

**Comment by chheis** [comment]
@lukasmittag  is this issue already fixed? As merge of #95 is already merged

**Comment by SebastianSchildt** [comment]
It is
